src/Platform.Handlers/samples/Sample/Controls/BoxView.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Xamarin.Forms;
+using Xamarin.Platform;
+
+namespace Sample
+{
+	public class BoxView : Xamarin.Forms.View, IBox
+	{
+		public Color Color { get; set; }
```

Comment:
@mattleibow @jsuarezruiz  Do we even need this property?

How does this property differ from setting `BackgroundColor` on the IView?

https://github.com/xamarin/Xamarin.Forms/blob/a01f00590694d9d20cb4ee40d783e81f2bb4c881/src/Platform.Handlers/src/Xamarin.Platform.Handlers/Core/IFrameworkElement.cs#L9

Like on a current BoxView in Forms what's different when I set Color vs BackgroundColor for a BoxView?

Reply:
I agree this property is not useful. Might even be detrimental for UX. Which one wins? If we are going to do gradients or images or something, then now we got this property in the way.

I vote to remove it. The forms layer can add it back for compact if need be. But on a base handler level, this is not useful.